comic_identity_engine/jobs/tasks.py

```
@@ -0,0 +1,857 @@
+                        )
+                        issues_created += 1
+
+                    if issue_candidate.variant_suffix:
+                        try:
+                            await variant_repo.create(
+                                issue_id=issue.id,
+                                variant_suffix=issue_candidate.variant_suffix,
+                            )
+                            variants_created += 1
+                        except Exception:
+                            pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)